extra-checkstyle-suppressions.xml

```diff
@@ -21,4 +21,8 @@
 <suppressions>
     <!-- Exclude auto-generated code -->
     <suppress checks=".*" files="target/routes/main/controllers/routes.java"/>
+
+    <!-- Exclude large nested external data models -->
+    <suppress checks=".*" files="app/com/arpnetworking/kairos/client/models/MetricsQuery.java"/>
```

Comment:
nit: why turn off all checks instead of just... `FileLength`, or whichever (presumably few) checks are actually going off?

Reply:
Good catch! That's just a copy-pasta error.